### PR TITLE
BCDA-3798 Updates page

### PR DESCRIPTION
**Pull request by PatKelsh**
### Fixes [BCDA-3798](https://jira.cms.gov/browse/BCDA-3798)

Added the new update page sections with guide on how to create future updates

### Security Implications

- [ ] new software dependencies

- [ ] security controls or supporting software altered

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

- [x] no PHI/PII is affected by this change

### CODE REVIEW

File: updates.html [hunk 1]
```diff
@@ -3,4 +3,19 @@
...
+    </div>
+    <div class="ds-l-col--12 ds-u-text-align--right ds-u-font-size--h1 ds-u-color--white">
+      Keep up with the latest news from BCDA
+    </div>
+  </div>
+  <div class="ds-l-col--12 ds-u-padding-y--2">
+    {% include updates/intro.html %}
+  </div>
+  <div class="ds-u-padding-y--2">
+    {% include updates/2020-10-28.html %}
```
Comment (mtrang1263) [inline updates.html:19]: Is it possible to have a wildcard used so we can include all content inside the `updates/` directory?

File: _includes/updates/2020-10-28.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<div class="ds-u-padding-y--2">
```
Comment (mtrang1263) [inline _includes/updates/2020-10-28.html:1]: Can we add an additional update example to see how the page looks with multiple entries?

Comment (PatKelsh) [reply _includes/updates/2020-10-28.html:1]: @mtrang1263 added another update to the page

**Review by mtrang1263 — APPROVED** [review]
Follow up PR will be created to resolve any outstanding comments

**Pull request merged**